CMakeLists.txt

```diff
@@ -443,21 +443,25 @@ install (TARGETS helics-static EXPORT helics-targets
     LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
 	set(helics_static_file "${CMAKE_STATIC_LIBRARY_PREFIX}helics-static${CMAKE_STATIC_LIBRARY_SUFFIX}")
-
+	set(helics_static_file_debug "${CMAKE_STATIC_LIBRARY_PREFIX}helics-static${CMAKE_DEBUG_POSTFIX}${CMAKE_STATIC_LIBRARY_SUFFIX}")
```

Comment:
Shouldn't it build the debug version only if the debug flag is set?

Reply:
this isn't building anything it is just setting a variable with the name.  The name gets used in a configure file for importing the exported targets.  If the debug version wasn't built there is another check in those files that clear an appropriate variable.  There were just things that weren't working very well with the import depending on the versions built,  this cleaned up some of that.

Reply:
when you install helics,  take a look in the cmake directory,  you will see a few cmake files that can be used to import all the helics libraries into another project.  This is how I do it for helics-fmi and GridDyn.

Reply:
Okay, my bad. This looks good then.